src/Symfony/Component/Config/Tests/Definition/BaseNodeTest.php

```diff
@@ -36,7 +36,32 @@ public function testGetPathForChildNode(string $expected, array $params)
             }
         }
 
-        $node = $this->getMockForAbstractClass(BaseNode::class, $constructorArgs);
+        $node = new class(...$constructorArgs) extends BaseNode {
+            protected function validateType($value): void
+            {
+            }
+
+            protected function normalizeValue($value)
+            {
+            }
+
+            protected function mergeValues($leftSide, $rightSide)
+            {
+            }
+
+            protected function finalizeValue($value)
+            {
+            }
```

Comment:
Those methods need to return something. Otherwise we won't be able to ad a return type after merging up.

```suggestion
            protected function normalizeValue($value)
            {
                return null;
            }

            protected function mergeValues($leftSide, $rightSide)
            {
                return null;
            }

            protected function finalizeValue($value)
            {
                return null;
            }
```

Reply:
Given that they return mixed, I think it would still without the return statement? I added return statement anyway, better be safe than sorry 🙂

Reply:
I should also add return types to other anonymous classes, I'm on it.

Reply:
Better now 👍

Reply:
> Given that they return `mixed`, I think it would still work at up-merge without the return statement?

Yes, but whoever does the merge-up will have to think less when doing so. 😅 Also, the diff is smaller.